Cue/app/tabs/library/TagsTableRow.js

```diff
@@ -0,0 +1,91 @@
+// @flow
+
+'use strict'
+
+import React from 'react'
+import { View, Text, TextInput, Platform } from 'react-native'
+
+import Chip from '../../common/Chip'
+import CueColors from '../../common/CueColors'
+import TableRow from '../../common/TableRow'
+
+const styles = {
+  newTagInput: {
+    fontSize: 14,
+    width: 120,
+    height: Platform.OS === 'ios' ? 28 : undefined,
+    marginVertical: Platform.OS === 'ios' ? 4 : undefined,
+  },
+  row: {
+    flexWrap: 'wrap',
+    paddingVertical: Platform.OS === 'ios' ? 8 : undefined,
+  },
+}
+
+type Props = {
+  tags: Array<string>,
+  disabled?: boolean,
+  onTagAdded: (tag: string) => void,
+  onTagRemoved: (tag: string) => void,
+}
+
+export default class TagsTableRow extends React.Component {
```

Comment:
You mean the class name? It's named this way since it displays in the style of a table row, which is based loosely off the [grouped `UITableView` style in iOS](https://developer.apple.com/ios/human-interface-guidelines/ui-views/tables/).

I'll add the limit to the `_onTagAdded` callback in `DeckSharingOptions`.